package/network/utils/layerscape/fmc/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2017 NXP
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fmc
+PKG_SOURCE_DATE:=2017-09-22
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/qoriq-open-source/fmc.git
+PKG_SOURCE_VERSION:=8c9f127138dc803bf58bef75e9fc092dc60f9c53
+PKG_MIRROR_HASH:=199b91c0a49ab4a4aa1eded7703ea1c11241e22d21a41f00fe992c043b28b467
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/fmc
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@TARGET_layerscape +libxml2 +tclap +fmlib +libstdcpp
+  TITLE:=Layerscape Frame Manager Configuration Tool
```

Comment:
build test fails as libxml2 is not part of the base feed. shall we drop the fmc stuff from the PR for now and merge the rest ?

Reply:
@blogic 
Sorry for my late response. I just come back to office from Chinese New Year holidays today.
And thanks a lot for your reveiwing.

So what should I do, if I want to add fmc support? Move libxml to base feed?
Thanks.

Reply:
Any suggestion? Actually fmc is an important tool for customer :)

Reply:
@hauke 
Also add Hauke for some suggestion :)

Hi Hauke,
Any idea for this problem? Thanks a lot.